react/components/SelectionListOrderBy.js

```diff
@@ -37,39 +51,34 @@ class SelectionListOrderBy extends Component {
     })
   }
 
+  getOrderByTitle = order => {
+    const { options, intl } = this.props
+    let title = intl.formatMessage({ id: 'search-result.orderby.title' })
+    if (location.search.includes('order')) {
+      options.forEach(opt => {
+        if (opt.value === order) {
+          title = opt.label
+        }
+      })
+    }
+    return title
+  }
+
   render() {
-    const { intl, options, orderBy } = this.props
+    const { orderBy } = this.props
+    const { showDropdown } = this.state
+    const btClass = `bg-base c-on-base t-action--small ml-auto ${showDropdown ? 'vtex-dropdown-btn__active b--muted-4' : 'vtex-dropdown-btn'}`
```

Comment:
the `vtex-dropdown-btn` class should always be present.

use the `classnames` package.

Reply:
also, rename it to `vtex-dropdown__btn`, follow the [BEM](https://en.bem.info/methodology/quick-start/#element) best practices.

Reply:
and the `vtex-dropdown-btn__active` to `vtex-dropdown__btn--active`